src/gen-ts.ts

```diff
@@ -234,6 +245,63 @@ function handleFunction(feature: AnalyzerFunction, root: ts.Document) {
 }
 
 /**
+ * Convert the given Analyzer properties to their TypeScript declaration
+ * equivalent.
+ */
+function handleProperties(analyzerProperties: Iterable<analyzer.Property>):
+    ts.Property[] {
+  const tsProperties = new Array<ts.Property>();
```

Comment:
nit: `new Array<ts.Property>()` -> `<ts.Property>[]`;

Reply:
Done. (I assume you mean `<ts.Property[]>[]`).